/**:
    ros__parameters:

        ## Device settings, provide one of the following:
        ##  - scan_for devices = 
        ##          true:   the driver ignores port and baudrate settings, scans for devices on all ports and selects the first found
        ##          false: = Requires correct port and baudrate both to be specified
        scan_for_devices: true
        port: "" # port name, e.g. '/dev/ttyUSB0'
        baudrate: 115200 # non necessary for some devices
        # config_baudrate: 921600

        ## Connect only to specific device_id:
        ##  device_id = '077007EF' (uppercase hex string), returns with error if the device with ID is not found on the ports
        ##  device_id = "" Null string disables this check
        device_id: "" # Null String ("") by default, else uppercase hex string, e.g. '077007EF'

        # Enable logging. Requires name of the file to be specified.
        enable_logging: false
        ## Log file (optional), placed in ~/.ros/ otherwise use absolute path
        log_file: log.mtb

        publisher_queue_size: 5

        # Configure the sensor to rotate its output using:
        alignment_local_quat: [ 1.0, 0., 0., 0. ] # (identity)
        alignment_sensor_quat: [ 1.0, 0., 0., 0. ] # (identity)

        # TF transform frame_id (default: imu_link), you may want to change it if you use multiple devices
        frame_id: "imu_link"

        # Message publishers
        pub_imu: true
        pub_quaternion: true
        pub_mag: true
        pub_angular_velocity: true
        pub_acceleration: true
        pub_free_acceleration: true
        pub_dq: true
        pub_dv: true
        pub_sampletime: true
        pub_temperature: true
        pub_pressure: true
        pub_gnss: true
        pub_twist: true
        pub_transform: true
        pub_positionLLA: true
        pub_velocity: true

        ## Sensor standard deviation [x,y,z] (optional)
        ## This value is used to override the covariance matrix in sensor_msgs/Imu and
        ## sensor_msgs/MagneticField messages.
        ## Important: Make sure the values provided are in decimal points, or else might give error in execution
        linear_acceleration_stddev: [0.0, 0.0, 0.0] # [m/s^2]
        angular_velocity_stddev: [0.0, 0.0, 0.0] # [rad/s]
        orientation_stddev: [0.0, 0.0, 0.0] # [rad]
        magnetic_field_stddev: [0.0, 0.0, 0.0] # [Tesla]

        # Configure the filter profile
        # onboard_filter_profile: "Robust/VRU"

        # Configure the output data
        # output_configuration:
        #     - "XDI_PacketCounter=65535"
        #     - "XDI_SampleTimeFine=65535"
        #     - "XDI_StatusWord=65535"
        #     - "XDI_Quaternion=200"
